keyboards/keaboard/rev1/config.h

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2023 Keith Wade <https://github.com/keawade>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 7
```

Comment:
This file should be removed.

Reply:
Removing this file breaks the compile.

```
Compiling: quantum/split_common/split_util.c                                                       quantum/split_common/split_util.c:19:10: fatal error: config.h: No such file or directory
 #include "config.h"
          ^~~~~~~~~~
compilation terminated.
 [ERRORS]
 | 
 | 
 | 
make[1]: *** [builddefs/common_rules.mk:360: .build/obj_keaboard_rev1_default/quantum/split_common/split_util.o] Error 1
Make finished with errors
make: *** [Makefile:414: keaboard:all] Error 1
```

Should I instead remove the matrix defines and leave the file?

Reply:
I believe this should only be an issue on the develop branch? Nonetheless, #19817 will fix it.

Reply:
I can't get it to build in my fork's branch even after syncing with QMK `master`. Is that expected? If so, is there a way I can easily get local builds working for local testing?